src/main/scala/org/ergoplatform/explorer/services/AddressesService.scala

```diff
@@ -27,7 +27,13 @@ class AddressesServiceIOImpl[F[_]](xa: Transactor[F], ec: ExecutionContext)
 
   override def getAddressInfo(addressId: String): F[AddressInfo] = for {
     _ <- Async.shift[F](ec)
-    info <- getAddressInfoResult(addressId)
+    info <- A.suspend{
+      if (addressId.startsWith("cd0703")) {
```

Comment:
could you add a comment here that the address is non-standard if it doesn't start from "cd0703"?